tests/e2e/tier-1.fmf

```diff
@@ -0,0 +1,25 @@
+summary: Tier 0 - QM Interconnect through bluechi
+
+discover:
+    how: fmf
+    filter: tier:1
+
+provision:
+   # Can not use provision no podman args
+   how: local
+
+adjust:
+   prepare+:
+      - name: Prepare Repos
+        how: shell
+        script: |
+           podman run --replace -d --name autosd --security-opt label=disable \
+                  --cap-add=sys_admin,mknod -it \
```

Comment:
```suggestion
                  --cap-add=sys_admin -it \
```

Reply:
Sure, let me ask the podman master, @rhatdan WDYT

Reply:
Currently there is another issue, I will stick to --privileged inside for now

Reply:
sure thing, could you please move this one to `Draft` until things get in the shape?